custom/conf/app.example.ini

```diff
@@ -1485,39 +1485,45 @@ PATH =
 ;; Prefix displayed before subject in mail
 ;SUBJECT_PREFIX =
 ;;
-;; Mail server
-;; Gmail: smtp.gmail.com:587
-;; QQ: smtp.qq.com:465
-;; As per RFC 8314 using Implicit TLS/SMTPS on port 465 (if supported) is recommended,
-;; otherwise STARTTLS on port 587 should be used.
-;HOST =
-;;
-;; Disable HELO operation when hostnames are different.
-;DISABLE_HELO =
-;;
-;; Custom hostname for HELO operation, if no value is provided, one is retrieved from system.
+;; Mail server protocol. One of "smtp", "smtps", "smtp+startls", "smtp+unix".
+;; If your provider does not explicitly say which protocol it uses but does provide a port,
+;; you can set SMTP_PORT instead and this will be inferred.
+;PROTOCOL =
+;
+;; Mail server address, e.g. smtp.gmail.com.
+;; For smtp+unix, this should be a path to a unix socket instead.
+;SMTP_ADDR =
```

Comment:
Generally L-G-T-M.

I prefer to use a URL to set all these protocol/addr/port. eg:

```
SMTP_ADDR = smtp+starttls://smtp.gmail.com:587
```

And IIRC there are some SMTP options on the installation page. Personally I prefer to remove these options from the installation page, but I don't know how others think about it .....

Reply:
Fair; I mostly just mirrored the HTTP options for my choices. In terms of the install page options, I just modified the existing ones to not break these settings and left them; I don't personally use the install page so I'm open to whatever people want to do there.